components/breakdown.py

```diff
@@ -55,12 +55,12 @@ def get_data(df, role_type="wave_role", overfill=200):
         value=200,
         step=100,
     )
-    df = load_tourney_results(league_to_folder[champ], result_cutoff=overfill)
+    df = load_tourney_results(league_to_folder[legend], result_cutoff=overfill)
```

Comment:
I won't change this component. Two reasons:
- you don't want to remove historical data from view
- this was fun in 2022-early 2023, now it's not really